docs/our-approach-to-data.md

```diff
@@ -377,48 +332,30 @@ export const itemsSchema = {
 };
 ```
 
-A JSON Schema must be provided if the subtree chooses to persist state. If we find that our persisted data doesn't
-match our described data shape, we should throw it out and rebuild that section of the tree with our default state.
-
-When using `createReducer` util you can pass a schema as a third param and all that will be handled for you.
-```javascript
-export const items = createReducer( defaultState, {
-	[THEMES_RECEIVE]: ( state, action ) => // ...
-}, itemsSchema );
-```
+In order to [opt-in](https://github.com/Automattic/wp-calypso/pull/13542) to persistence,
+A JSON Schema must be provided as a property of the reducer.
+If we find that the shape of the persisted data doesn't match the schema, we throw it out
+and rebuild that section of the tree with its default state.
 
-If you are not satisfied with the default handling, it is possible to implement your own `SERIALIZE` and
-`DESERIALIZE` action handlers in your reducers to customize data persistence. Always use a schema with your custom
-handlers to avoid data shape errors.
-
-### Opt-in to Persistence ( [#13542](https://github.com/Automattic/wp-calypso/pull/13542) )
-
-If we choose not to use `createReducer` we can opt-in to persistence by adding a schema as a property on the reducer.
 We do this by combining all of our reducers using `combineReducers` from `state/utils` at every level of the tree instead
 of [combineReducers](http://redux.js.org/docs/api/combineReducers.html) from `redux`. Each reducer is then wrapped with
-`withSchemaValidation` which returns a wrapped reducer that validates on `DESERIALIZE` if a schema is present and
-returns initial state on both `SERIALIZE` and `DESERIALIZE` if a schema is not present.
+`withSchemaValidation` which returns a wrapped reducer that validates on `DESERIALIZE`.
 
-To opt-out of persistence we combine the reducers without any attached schema.
+You can add a schema property to any reducer:
 ```javascript
-return combineReducers( {
-    age,
-    height,
+export const items = createReducer( defaultState, {
+	[THEMES_RECEIVE]: ( state, action ) => // ...
 } );
+items.schema = itemsSchema; // this will now be persisted
 ```
+Note: we used to encourage passing in `itemsSchema` as a third parameter to createReducer.
+This is now deprecated because it is less flexible than adding a schema directly to any reducer.
```

Comment:
Thanks for adding this note here! It's super important to tie historical context into current practices :smile: